requirements.txt

```diff
@@ -2,10 +2,8 @@ pandas==0.25.3
 scipy==1.4.1
 numpy==1.18.1
 scikit-learn==0.22.1
-pysqlite3==0.4.1
```

Comment:
Don't we still need pysqlite3?

Reply:
Is there a reason, we need

https://github.com/coleifer/pysqlite3

as we are only using build-in sqlite3 in cache.py?

Reply:
Good catch. Right you are.

Reply:
Done